src/test/resources/fixtures/createLegacyBeaconRequest.json

```
@@ -0,0 +1,150 @@
+          "tripInfo": "None",
+          "rigName": "None",
+          "beaconPosition": "None",
+          "position": "None",
+          "localManagementId": "None",
+          "beaconNsn": "None",
+          "beaconPartNumber": "None",
+          "notes": "None",
+          "pennantNumber": "None",
+          "aircraftDescription": "None",
+          "survivalCraftType": "None",
+          "communications": "VHF/DSC",
+          "isMain": "Y",
+          "createUserId": 2889,
+          "createDt": "2015-03-24T11:35:11",
+          "updateUserId": 2889,
+          "updateDt": "2015-03-24T11:35:11",
+          "versioning": 0,
+          "useType": "Maritime",
+          "vesselType": "Pleasure",
+          "aircraftType": "None",
+          "landUse": "None",
+          "note": "Some notes"
+        },
+        {
+          "pkBeaconUsesId": 6058,
+          "fkBeaconId": 6062,
+          "vesselName": "KAKEE",
+          "homePort": "None",
+          "maxPersons": 10,
+          "officialNumber": "None",
+          "rssSsrNumber": "None",
+          "callSign": "VQAS7",
+          "imoNumber": "None",
+          "mmsiNumber": 235007399,
+          "fishingVesselPln": "None",
+          "hullIdNumber": "None",
+          "cg66RefNumber": "None",
+          "aodSerialNumber": "None",
+          "principalAirport": "None",
+          "bit24AddressHex": "None",
+          "aircraftRegistrationMark": "None",
+          "areaOfUse": "None",
+          "tripInfo": "None",
+          "rigName": "None",
+          "beaconPosition": "None",
+          "position": "None",
+          "localManagementId": "None",
+          "beaconNsn": "None",
+          "beaconPartNumber": "None",
+          "notes": "None",
+          "pennantNumber": "None",
+          "aircraftDescription": "None",
+          "survivalCraftType": "None",
+          "communications": "VHF/DSC",
+          "isMain": "N",
+          "createUserId": 2889,
+          "createDt": "2015-03-24T11:35:11",
+          "updateUserId": 2889,
+          "updateDt": "2015-03-24T11:35:11",
+          "versioning": 0,
+          "useType": "Maritime",
+          "vesselType": "Pleasure",
+          "aircraftType": "None",
+          "landUse": "None",
+          "note": "Some more notes"
+        }
+      ],
+      "owner": {
+        "pkBeaconOwnerId": 231035,
+        "fkBeaconId": 6062,
+        "ownerName": "Mr Beacon",
+        "companyName": "Beacon house",
+        "careOf": "Sideshow beacon",
+        "address1": "Bristol",
+        "address2": "Beacons",
+        "address3": "Beek",
+        "address4": "Me",
+        "country": "UNITED KINGDOM",
+        "postCode": "BS9 6GF",
+        "phone1": "077123456786",
+        "phone2": "077123456787",
+        "mobile1": "077123456788",
+        "mobile2": "077123456789",
+        "fax": "Fax me",
+        "email": "beacons@beacons.com",
+        "isMain": "Y",
+        "createUserId": "3748",
+        "createdDate": "2021-07-27 17:15:24",
```

```suggestion
        "createdDate": "2021-07-27T17:15:24",
```
Do they need that to make em ISO format-y